pyhyp/utils.py

```diff
@@ -38,15 +40,39 @@ def simpleOCart(nearFile, dh, hExtra, nExtra, sym, mgcycle, outFile, userOptions
         Output file name.
 
     userOptions : dict, optional
-        The name of the nearfield CGNS file to mesh around.
+        Custom pyhyp options to be used with this extrusion.
+
+    xBounds : array (2 x 3)
```

Comment:
Can you clean up the docstrings for `xBounds` and `useFarfield` a bit?
The `xBounds` docstring is not rendering correctly: https://mdolab-pyhyp--66.com.readthedocs.build/en/66/API.html
I would avoid indentation and use backticks for the code snippets.

Other things to fix:
- `, optional` should go after the type (instead of in the description)
- capitalization and splitting up sentences